scripts/hudson/narayana.sh

```diff
@@ -438,6 +434,11 @@ function init_jboss_home {
   grep 'connection-timeout' "${CONF}"
   #Enable remote debugger
   echo JAVA_OPTS='"$JAVA_OPTS -agentlib:jdwp=transport=dt_socket,address=8787,server=y,suspend=n"' >> "$JBOSS_HOME"/bin/standalone.conf
+
+  if [ $AS_TESTS_TRACE ]; then
```

Comment:
This should allow other axis to benefit from being able to enable AS trace still